setup.py

```diff
@@ -36,10 +36,7 @@ def get_install_requires():
             continue
         # add line to requirements
         requirements.append(line.replace('\n', ''))
-    if sys.version_info.major >= 3:
         requirements.append("networkx>=2.0,<2.5")
```

Comment:
this should have been removed and `networkx>=2.0,<2.5` moved to the `requirements.txt` file, but I'll do that myself